### ISSUE TITLE
System.getConfig() not compatible with System.config(config)

### ISSUE THREAD

**Issue by mpfau** [systemjs/systemjs#1604]
When invoking System.config() with a configuration retrieved via System.getConfig(), the contents of the bundles are not initialized.

This seems to be the case because System.getConfig returns an object structure like:
``` js
bundles: {
    bundleA: {0: 'dependencyA', 1: 'dependencyB'}
}
```
However, System.config(config) expects the contents of a bundle to be describes as an array:
``` js
bundles: {
    bundleA: ['dependencyA', 'dependencyB']
}
```

**Comment by guybedford** [comment]
Thanks, I've added a fix in https://github.com/systemjs/systemjs/commit/c9cccd18d4066af40ef86b3c7cad38e6453715f1.

**Comment by guybedford** [comment]
Released in 0.20.10.
